src/runtime/arch/arm64-options.mk

```diff
@@ -19,3 +19,4 @@ FCJAILERCMD := jailer
 
 # cloud-hypervisor binary name
 CLHCMD := cloud-hypervisor
+CLHSNPCMD := cloud-hypervisor-snp
```

Comment:
These paths are the same but it follows the same pattern as QEMU, which may be necessary for upstreaming